test/factory/users.js

```diff
@@ -0,0 +1,13 @@
+const { factory } = require('factory-girl');
+
+const db = require('../../app/models');
+
+factory.define('User', db.User, {
+  firstName: 'Esteban',
+  lastName: 'Herrera',
+  email: 'esteban.herrera@wolox.co',
+  password: 'contrasena1234'
```

Comment:
Here you can use the [faker](https://www.npmjs.com/package/faker) library to create fake data, so it is not the same user always

Reply:
Done. Let the email and password fixed because the password generated by faker on some occasions does not comply with the established conditions and I need the email to be the same for the test "email in use".

Reply:
The password is fine, but I think you can specify the type of string you want, so it always returns alphanumeric.
The email should be generated randomly, so u can more easily use the factory-girl functions for example: 
```
module.exports = {
  create: params => factory.create('user', params),
  createMany: () => factory.createMany('user', 5),
  build: params => factory.build('user', params),
  attributes: params => factory.attrs('user', params)
};
```
The createMany() function would be useful later, but you can't have every user with the same email